test/tests/ver-1fc/ver-1fc.i

```diff
@@ -0,0 +1,171 @@
+T_SA = 600 # K
+T_SB = 0 # K
+L_A = 40e-2 # m
+L_B = 40e-2 # m
+k_A = 401 # W/m/K
+k_B = 80.2 # W/m/K
+num_nodes = 800 # (-)
```

Comment:
Is this meant to mean "no units"? It took me a second to parse why something clearly named `num_nodes` would need to have a units designation.

Reply:
Yes, it's what's commonly used for no units. I just think it doesn't hurt and establishes a standard to always think about units.